### DISCUSSION TITLE
How to call/access CSLA class having Authorisation added by AddObjectAuthorizationRules method

### DISCUSSION THREAD

**Discussion by vinayiscool** [MarimerLLC/csla#2081] [Questions]
I am facing issue while executing update command which internally first calling fetch operation of that command related **object** and i am passing identity with required role of that **object** which i am using for Fetch operation, even its saying User is not allowed to access object.

Can you please help me on that or how should i access internal fetch object used in updatecommand ( Csla.DataPortal.ExecuteAsync<Updatecommand>(command)) ?

@rockfordlhotka

**Comment by rockfordlhotka** [comment]
Can you share your code?
